obelisk_ws/src/obelisk_ros/launch/obelisk_bringup.launch.py

```diff
@@ -94,7 +94,8 @@ def obelisk_setup(context: launch.LaunchContext, launch_args: Dict) -> List:
         )
     )  # when the global state node enters its shutdown state, the launch file also shuts down
     obelisk_launch_actions += [global_state_node, shutdown_event_handler]
-    if auto_start:
+    if auto_start in ("true", "activate"):
```

Comment:
mega nit: throughout this repo, i use `if _ in [option1, option2, ...]` (list instead of tuple). either is actually fine, but just for stylistic consistency, change it.